src/gui/SetupTotpDialog.cpp

```diff
@@ -0,0 +1,74 @@
+#include "SetupTotpDialog.h"
```

Comment:
@louib Is there a copyright header template somewhere I can follow? Or can I just copy it from existing files and put my name?